File: Common/Entities/TestRules.cs
using System;
using System.Collections.Generic;
using System.Text;
using Common.BusinessObjects;

namespace Common.Entities
{
	public class TestRules
	{
		public int Id { get; set; }

		public int RulesSectionId { get; set; }

		public RulesSection RulesSections { get; set; }

		public int DrivingTestId { get; set; }

		public DrivingTest DrivingTest { get; set; }
	}
}

File: AutoSchool/ClientApp/src/app/shared/entities/TestRules.ts
import { RuleSection } from './RuleSection';
import { DrivingTest } from './DrivingTest';


export class TestRules {
    constructor(
        public id?: number,
        public ruleSectionId?: number,
        public ruleSection?: RuleSection,
        public drivingTestId?: number,
        public drivingTest?: DrivingTest
    ) { }
}
